index.html

```diff
@@ -1292,10 +1292,11 @@
 
 ### Ancillary uses
 
-[=Sites=] sometimes use data in ways that aren't needed for the user's immediate
+[=Sites=] sometimes use data in ways that aren't needed for the user's primary
 goals. For example, they might bill advertisers, measure site performance, or
-tell developers about bugs. These uses are known as <dfn data-lt="ancillary
-use">ancillary uses</dfn>.
+tell developers about bugs. These uses, which provide indirect support for,
```

Comment:
```suggestion
tell developers about bugs. These [=ancillary uses=], which provide indirect support for,
```

If you need a connection between paragraphs, this might do the job.

Reply:
You're removing that sentence in your suggested changes, though, so you can't add markup to it.

Reply:
Oh, that's right, Let me try again.